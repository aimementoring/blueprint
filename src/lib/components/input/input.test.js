import React from 'react';
import renderer from 'react-test-renderer';
import Input from './input';

describe('Input', () => {
  it('renders properly', () => {
    const component = (
      <Input placeholder="Name" value="" name="name" type="text" required disabled />
    );
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
