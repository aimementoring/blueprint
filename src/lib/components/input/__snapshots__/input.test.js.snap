// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders properly 1`] = `
<div
  className="theme-purple"
>
  <div
    className="theme-purple"
  >
    <div
      className="inputWrapper "
    >
      <input
        className=" input 
              false"
        disabled="disabled"
        name="name"
        onChange={[Function]}
        placeholder="Name"
        required={true}
        type="text"
        value=""
      />
    </div>
  </div>
  
</div>
`;
